### PR TITLE
DOCAPI-6213: Docs for send_progress_in_http_headers setting.

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation

### CODE REVIEW

File: docs/en/interfaces/http.md [hunk 1]
```diff
@@ -205,7 +205,13 @@ For information about other parameters, see the section "SET".
...
 
-You have the option to receive information about the progress of query execution in X-ClickHouse-Progress headers. To do this, enable the setting send_progress_in_http_headers.
+You have the option to receive information about the progress of query execution in `X-ClickHouse-Progress` headers. To do this, enable the setting [send_progress_in_http_headers](../operations/settings/settings.md#settings-send_progress_in_http_headers). Example of the header:
+
+```
+X-ClickHouse-Progress: {"read_rows":"393216","read_bytes":"28368934","total_rows":"8880128"}
+```
+
+File [Progress.h](https://github.com/yandex/ClickHouse/blob/master/dbms/src/IO/Progress.h) contains structures `struct ProgressValues` and `struct Progress` which define fields that you see in the HTTP header.
```
Comment (blinkov) [inline docs/en/interfaces/http.md]: why not just document them here?

File: docs/en/interfaces/http.md [hunk 2]
```diff
@@ -205,7 +205,13 @@ For information about other parameters, see the section "SET".
 
 Similarly, you can use ClickHouse sessions in the HTTP protocol. To do this, you need to add the `session_id` GET parameter to the request. You can use any string as the session ID. By default, the session is terminated after 60 seconds of inactivity. To change this timeout, modify the `default_session_timeout` setting in the server configuration, or add the `session_timeout` GET parameter to the request. To check the session status, use the `session_check=1` parameter. Only one query at a time can be executed within a single session.
 
-You have the option to receive information about the progress of query execution in X-ClickHouse-Progress headers. To do this, enable the setting send_progress_in_http_headers.
+You have the option to receive information about the progress of query execution in `X-ClickHouse-Progress` headers. To do this, enable the setting [send_progress_in_http_headers](../operations/settings/settings.md#settings-send_progress_in_http_headers). Example of the header:
```
Comment (blinkov) [inline docs/en/interfaces/http.md]: response headers

File: docs/en/interfaces/http.md [hunk 3]
```diff
@@ -205,7 +205,13 @@ For information about other parameters, see the section "SET".
...
+
+```
+X-ClickHouse-Progress: {"read_rows":"393216","read_bytes":"28368934","total_rows":"8880128"}
```
Comment (blinkov) [inline docs/en/interfaces/http.md]: probably the example could demonstrate how these values change over time for a single query

File: docs/en/operations/settings/settings.md [hunk 4]
```diff
@@ -114,6 +114,19 @@ Possible values:
 
 Default value: 0.
 
+## send_progress_in_http_headers {#settings-send_progress_in_http_headers}
+
+Enables or disables sending of the `X-ClickHouse-Progress` header in an answer from `clickhouse-server`.
```
Comment (blinkov) [inline docs/en/operations/settings/settings.md]: header -> HTTP response headers

**Review by blinkov — CHANGES_REQUESTED** [review]

**Comment by BayoNet** [comment]
Тут вроде всё хорошо. Вмёржить бы.

**Pull request merged**